src/utils/config.ts

```diff
@@ -114,6 +151,9 @@ const errorOrConfig: t.Validation<IConfig> = IConfig.decode({
   ENABLE_ADE_AA: fromNullable(process.env.ENABLE_ADE_AA)
     .map(_ => _.toLowerCase() === "true")
     .getOrElseL(() => false),
+  ENABLE_FULL_OPERATOR_METADATA: fromNullable(process.env. ENABLE_FULL_OPERATOR_METADATA)
```

Comment:
Sorry, a whitespace typo in the previous code suggestion.
```suggestion
  ENABLE_FULL_OPERATOR_METADATA: fromNullable(process.env.ENABLE_FULL_OPERATOR_METADATA)
```

Reply:
Take a look at the linter error in the review pipeline.